### PR TITLE
Add on_premises_immutable_id column to azuread_user table

### PR DESCRIPTION
**Pull request by maceip**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
steampipe query "select on_premises_immutable_id from azuread_user"
+--------------------------+
| on_premises_immutable_id |
+--------------------------+
| <null>                   |
| AADBackdoor              |
+--------------------------+

```
</details>

### CODE REVIEW

**Comment by rajlearner17** [comment]
Hi, @maceip thanks for using Steampipe. great to see this PR. We are looking into it and update you soon. 👍

File: azuread/table_azuread_user.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ func tableAzureAdUser() *plugin.Table {
 			{Name: "filter", Type: proto.ColumnType_STRING, Transform: transform.FromQual("filter"), Description: "Odata query to search for resources."},
 
 			// Other fields
+			{Name: "on_premises_immutable_id", Type: proto.ColumnType_STRING, Description: "Used to associate an on-premises Active Directory user account with their Azure AD user object."},
```
Comment (bigdatasourav) [inline azuread/table_azuread_user.go:56]: @maceip Could you please sort the columns in `Other fields` section. `on_premises_immutable_id` column should be after `mail_nickname` column.

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Comment by misraved** [comment]
Hello @maceip, once again thanks for your contribution 👍 

This feature is now available on `azuread v0.1.0` (https://hub.steampipe.io/plugins/turbot/azuread).

Please let us know how your experience has been so far and if we could help in case of any blockers 👍

**Pull request merged**